e2e/actions/l1_miner.go

```diff
@@ -165,6 +165,12 @@ func (s *L1Miner) ActL1EndBlock(t Testing) {
 	}
 }
 
+func includeL1Block(t StatefulTesting, miner *L1Miner, sender common.Address) {
```

Comment:
I think we can make this like below:
```suggestion
func (s *L1Miner) includeL1Block(t StatefulTesting, sender common.Address) {
```

Reply:
Please apply this comment :)